strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/repository/NugetRepositoryFeatures.java

```diff
@@ -104,8 +104,8 @@ public void downloadRemoteFeed(String storageId,
                                                             .orElse(REMOTE_FEED_PAGE_SIZE);
         for (int i = 0; true; i++)
         {
-            if (!downloadRemoteFeed(storageId, repositoryId, filter, searchTerm, targetFramework,
-                                    i * remoteFeedPageSize, remoteFeedPageSize))
+            if (!downloadRemoteFeed(storageId, repositoryId, nugetSearchRequest, i * remoteFeedPageSize,
+                                    remoteFeedPageSize))
```

Comment:
Cosmetic: When multi-line, please always chop the methods, or parameters to one per line. Thanks! :)

Reply:
@carlspring I need to dig deeply into Eclipse formatter settings for this, will do it,  but can we leave this as is within this PR?